share/extender/build_input.yml

```diff
@@ -82,7 +82,7 @@ platforms:
       # C++
       defines:          ['DDF_EXPOSE_DESCRIPTORS']
       cxxFlags:         []
-      cxxShFlags:       []
+      cxxShFlags:       ['-fno-rtti']
```

Comment:
Needed to link with the rive libraries

Reply:
Could this have any unwanted side-effects when building other extensions?